### DISCUSSION TITLE
[FEATURE REQUEST] ssl_version PROTOCOL_TLSv1_3

### DISCUSSION THREAD

**Discussion by Gunni** [saltstack/salt#67336] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
TLS 1.3 support is missing.

**Describe the solution you'd like**
Add TLS 1.3 support.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Please Note**
Minor addition, doesn't break anything by adding it since your config hard-codes a specific version, instead of a min version.

p.s. Maybe add min_version as an option too?

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://via.vmw.com/salt-slack)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!
